### PR TITLE
Fixed overlapping of boxes

### PR DESCRIPTION
**Pull request by darkfated**
It was
image
Become
image

### CODE REVIEW

**Comment by FPtje** [comment]
The boxes shouldn't be overlapping even before the pull request. Could there be some sort of font issue on your device that causes the box heights to be different for you?

Also, what platform do you run gmod on? Windows, Linux, OSX?

**Comment by DBotThePony** [comment]
if you do this PR, then change code to properly use [draw.GetFontHeight](https://wiki.facepunch.com/gmod/draw.GetFontHeight) to fix ANY possible issues with height of boxes

**Comment by darkfated** [comment]
Now no one will have any problems

**Comment by darkfated** [comment]
Inattention :)

File: gamemode/modules/chat/cl_chatlisteners.lua [hunk 1]
```diff
@@ -44,8 +44,9 @@ Draw the results to the screen
 local function drawChatReceivers()
     if not receivers then return end
 
+    local fontHeight = draw.GetFontHeight( 'DarkRPHUD1' )
```
Comment (FPtje) [inline gamemode/modules/chat/cl_chatlisteners.lua:47]: Could you fix the glualint warnings? Thanks!

**Comment by darkfated** [comment]
Done. Grateful for the tip

**Comment by FPtje** [comment]
Huh, I just installed GMod on Linux, and I can actually reproduce the overlapping boxes. Thanks for the fix! I'll test on both Windows and Linux to be sure.

**Comment by FPtje** [comment]
Works on both. Thanks for finding and solving this issue! Much appreciated.

**Pull request merged**